recipes/mutamr/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mutamr" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "6ab552dc1087bd88b1dcd58c71e52a33f3338e4cf28ffc39d9e096c060075816" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - mutamr=mutamr.mutamr:main
+  run_exports:
+    - {{ pin_subpackage('mutamr', max_pin="x.x") }}
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+  run:
+    - python >=3.10
+    - bwa =0.7.18
+    - delly =1.2.8
+    - freebayes =1.3.8
+    - bcftools =1.20
+    - samtools =1.20
+    - samclip =0.4.0
+    - snpeff =5.2
```

Comment:
Are such strict pins _absolutely_ necessary?

Reply:
I believe so - in order to ensure expected behaviour these dependencies have been tested and are 1. compatible (cause no clashes or other issues upon installation) and 2. ensure that the resulting files generated are as expected